packages/phone-number-privacy/migrations/20200330212224_create-accounts-table.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from 'knex'
+
+export async function up(knex: Knex): Promise<any> {
+  return knex.schema.createTable('accounts', (t) => {
+    t.increments('id')
+      .unsigned()
+      .primary()
+    t.dateTime('createdAt').notNullable()
+    t.string('address').notNullable()
```

Comment:
curious -- any reason the address itself can't be the primary key?

Reply:
Yep we could do that, makes sense